.changeset/cuddly-chairs-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-app-api': minor
```

Comment:
```suggestion
'@backstage/backend-app-api': patch
```

since it's an input type that people are not meant to implement themselves, i think it's fine to consider this a feature add